eclipse/src/saros/Saros.java

```diff
@@ -260,29 +268,34 @@ private void stopLifeCycle() {
   }
 
   private void setupLoggers() {
-    /*
-     * HACK this is not the way OSGi works but it currently fulfill its
-     * purpose
-     */
-    final ClassLoader contextClassLoader = Thread.currentThread().getContextClassLoader();
 
     final boolean isDebugMode = Boolean.getBoolean("saros.debug") || isDebugging(); // $NON-NLS-1$
 
-    final String log4jPropertyFile =
-        isDebugMode
-            ? "saros_debug.log4j.properties" //$NON-NLS-1$
-            : "saros_release.log4j.properties"; //$NON-NLS-1$
-
     try {
-      // change the context class loader so Log4J will find the appenders
-      Thread.currentThread().setContextClassLoader(Saros.class.getClassLoader());
+      File mainLogDir = getStateLocation().toFile();
```

Comment:
Guess you took the naming from IntelliJ. It is not the "mainLogDir" but just a directory assigned by Eclipse to store plugin information for the current workspace.

Reply:
I will rename it to "pluginLogDir"